pandas/plotting/_matplotlib/misc.py

```diff
@@ -1,6 +1,12 @@
 # being a bit too dynamic
+from math import pi, sqrt
```

Comment:
You can delete the comment, I guess it refers to how we were importing before. But doesn't seem to add much value in any case.

@jreback do you want to avoid loading `math` unless the user calls the andrews_curves plot, like it was? Or you're ok with this? same for random below

Reply:
change these to be np.pi and np.sqrt; we don't use math generally